packages/earth-map/src/pages/profile/component.tsx

```diff
@@ -48,6 +48,7 @@ export function ProfileComponent(props: IProps) {
   });
   const [resetPasswordState, setResetPasswordState] = useState(RESET_PASSWORD_STATE.INITIAL);
   const [markedOrgsForLeave, setMarkedOrgsForLeave] = useState({});
+  const [hasLeftOrg, setHasLeftOrg] = useState(false);
```

Comment:
please add a comment here about why we have this state and also a "TODO: remove this when we find a way to do a full reset (redux and auth0 context) and use redux-first-router link"